### ISSUE TITLE
yolo11n：train loss increase， val loss decrease， and map increase suddenly

### ISSUE THREAD

**Issue by shihuiyu** [ultralytics/ultralytics#17649]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

Hi. I trained detection on COCO dataset with yolo11-n cfg file. Parameters were agree with 'default.yaml' except batchsize.
Total train epoch were 300. 
During the training, it was normal, except after the 287 epoch. Suddenly train loss increased, val loss decreased, mAP also went up. I adjusted the channel number and data dimension of the network for several times to retrain, and this problem occurred all the time,  each time a mutation occurred at the position of 287epoch.
![2024-11-20 142247](https://github.com/user-attachments/assets/e9fe3af8-3bde-4ba0-b568-d24fb2ea7a2b)

### Environment

New https://pypi.org/project/ultralytics/8.3.18 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.13 🚀 Python-3.10.15 torch-2.4.1+cu121 CUDA:0 (NVIDIA A100-PCIE-40GB, 40326MiB)

### Minimal Reproducible Example

It is your code.

### Additional

none

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @shihuiyu, thank you for bringing this to our attention 🚀! This is an automated response, and an Ultralytics engineer will get back to you soon.

We appreciate your detailed report which helps us understand the issue better. To assist us further in diagnosing this 🐛, could you provide a more specific [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/)? This will greatly aid in pinpointing the problem.

In the meantime, please ensure that you are using the latest version of the `ultralytics` package. You can update it by running:

```bash
pip install -U ultralytics
```

Additionally, we recommend reviewing our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure that all training settings are optimal.

Join the vibrant Ultralytics community for more interactive help and discussions. You can connect on [Discord](https://ultralytics.com/discord) 🎧, participate in in-depth conversations on [Discourse](https://community.ultralytics.com), or join our [Subreddit](https://reddit.com/r/ultralytics).

## Verified Environments

To see if the issue persists in different settings, consider trying out these verified environments with all the necessary dependencies:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing, ensuring that YOLO is operating correctly across various platforms.

Thank you for your patience and for being part of our community!

**Comment by Y-T-G** [comment]
The last 10 epochs disable mosaic augmentation, so change in losses is expected. You can change this behavior by passing `close_mosaic`. Default is 10, so last 10 epochs.

https://docs.ultralytics.com/modes/train/#train-settings

**Comment by shihuiyu** [comment]
> The last 10 epochs disable mosaic augmentation, so change in losses is expected. You can change this behavior by passing `close_mosaic`. Default is 10, so last 10 epochs.
> 
> https://docs.ultralytics.com/modes/train/#train-settings

Thanks for your reply.
When i trained on my custom dataset for 100 epochs, train loss decrease at last 10 epochs and the mAP has not  big change.
But on COCO dataset, the train loss increase too much. 
Is it normal?

**Comment by shihuiyu** [comment]
> The last 10 epochs disable mosaic augmentation, so change in losses is expected. You can change this behavior by passing `close_mosaic`. Default is 10, so last 10 epochs.
> 
> https://docs.ultralytics.com/modes/train/#train-settings

Thanks for your reply.
When i trained on my custom dataset for 100 epochs, train loss decrease at last 10 epochs and the mAP has not  big change.
But on COCO dataset, the train loss increase too much. 
Is it normal?

**Comment by shihuiyu** [comment]
> The last 10 epochs disable mosaic augmentation, so change in losses is expected. You can change this behavior by passing `close_mosaic`. Default is 10, so last 10 epochs.
> 
> https://docs.ultralytics.com/modes/train/#train-settings

Thanks for your reply.
When i trained on my custom dataset for 100 epochs, train loss decrease at last 10 epochs and the mAP has not big change.
But on COCO dataset, the train loss increase too much.
Is it normal?
![2024-11-21 100548](https://github.com/user-attachments/assets/650dea03-6034-4b97-86fa-510a912cbaec)

**Comment by glenn-jocher** [comment]
@shihuiyu it's common for the train loss to increase when mosaic augmentation is disabled in the last 10 epochs, especially on complex datasets like COCO. This is because the model is exposed to less varied data, allowing it to focus on final adjustments. If mAP is stable or improving, this behavior is typically not concerning. For more details on training settings, you can refer to the [Ultralytics training documentation](https://docs.ultralytics.com/modes/train/#train-settings).

**Comment by shihuiyu** [comment]
> @shihuiyu it's common for the train loss to increase when mosaic augmentation is disabled in the last 10 epochs, especially on complex datasets like COCO. This is because the model is exposed to less varied data, allowing it to focus on final adjustments. If mAP is stable or improving, this behavior is typically not concerning. For more details on training settings, you can refer to the [Ultralytics training documentation](https://docs.ultralytics.com/modes/train/#train-settings).

Thanks for reply.
